state/integration_test.go

```diff
@@ -54,6 +54,9 @@ func (s *IntegrationTestSuite) SetupSuite() {
 
 	accessor, err := NewCoreAccessor(s.cctx.Keyring, accountName, localHeader{s.cctx.Client}, nil, "")
 	require.NoError(s.T(), err)
+	ctx, cancel := context.WithCancel(context.Background())
+	accessor.ctx = ctx
+	accessor.cancel = cancel
```

Comment:
Why not Start?

Reply:
Start has more wider logic that we need in the test suite. But need only `stakingCli` and `abciQueryCli`. Also, I'm not aware how test suite behaves on app side after we call Start.